assets/src/application/ui/input/EspressoButton/Tooltip.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classNames from 'classnames';
+import { isMobile } from 'react-device-detect';
+import { Tooltip as DefaultTooltip } from 'antd';
+import { TooltipPropsWithTitle } from 'antd/lib/tooltip';
+import { __ } from '@wordpress/i18n';
+
+import { isEmpty } from '@appServices/utilities/string';
+
+interface TicketsChainedButtonProps extends TooltipPropsWithTitle {
+	button: JSX.Element;
+	buttonText?: string;
+	tooltip?: string;
+}
+
+const Tooltip: React.FC<TicketsChainedButtonProps> = ({ button, buttonText, tooltip, ...props }) => {
+	const mobileTooltip = isEmpty(buttonText) ? (
+		<div>
+			{button}
+			<div className='esprs-button__tooltip'>{tooltip}</div>
+		</div>
```

Comment:
if you give this wrapper div a classname then you could use flexbox to position things inside it

Reply:
done, thx